.golangci.yaml

```diff
@@ -3,6 +3,7 @@ run:
   build-tags:
   - integration_tests
   - e2e_tests
+  - experimental
```

Comment:
🤔 Does that mean we won't check `test/conformance/gateway_conformance_test.go` ?

Reply:
Unfortunately yes and without it won't check `test/conformance/gateway_expermimental_conformance_test.go`. And there is no nice way to resolve it 😞 https://github.com/golangci/golangci-lint/issues/1646
To consider wherever we have built tag add tag `all || ` and now by specifying one tag everything will be included, but it may have some downsides... To consider

Reply:
Given that we have particular test types in separate directories anyway I'm wondering now if using build tags for this purpose makes sense 🤔

Reply:
Alright got it. My suggestion then would be to change this into a test flag so that we can control which tests are run this way and get rid of the `exprimental` build tag. WDYT?

Reply:
When I worked on this, I used release candidate and it forced me to do it this way, but now I see 
- https://github.com/kubernetes-sigs/gateway-api/issues/2242

It's been removed in `v0.8.0`, thus I can rework it and get rid of it

Reply:
done @pmalek